packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js

```diff
@@ -49,6 +55,7 @@ export default class PciInstancesAddController {
     atInternet,
     PciProject,
     OvhApiCloudProjectRegion,
+    PciPrivateNetworksAdd,
```

Comment:
**change request** : private networks has been migrated, but the clean-up task has not yet been performed, which is why it works. But as soon as the private networks angularjs code is removed, instance creation will break. Please import the code you need and add it to the instance creation service instead